src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -340,3 +361,27 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig replica create') as c:
         c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location at which to create the replica.')
+
+    with self.argument_context('appconfig snapshot create') as c:
+        c.argument('snapshot_name', arg_type=snapshot_name_arg_type)
+        c.argument('filters', arg_type=snapshot_filter_arg_type)
+        c.argument('composition_type', arg_type=get_enum_type(["key", "key_label"]), help='Composition type used in building app configuration snapshots. If not specified, defaults to key.')
+        c.argument('retention_period', type=int, help='Duration in seconds for which a snapshot can remain archived before expiry. A snapshot can be archived for a maximum of 7 days (604,800s) for free tier stores and 90 days (7,776,000s) for standard tier stores. If specified, retention period must be at least 1 hour (3600s)')
+        c.argument('tags', arg_type=tags_type, help="Space-separated tags: key[=value] [key[=value] ...].")
+
+    with self.argument_context('appconfig snapshot show') as c:
+        c.argument('snapshot_name', arg_type=snapshot_name_arg_type)
+        c.argument('fields', arg_type=snapshot_fields_arg_type)
+
+    with self.argument_context('appconfig snapshot list') as c:
+        c.argument('snapshot_name', options_list=['--snapshot-name', '-s'], help='If no name specified, return all snapshots by default. Support star sign as filters, for instance abc* means snapshots with abc as prefix to the name.')
```

Comment:
Do we support any type of * regex here, or just prefix matching?

Reply:
It is just prefix matching. Would it be better to re-word `Support star sign as filters` to `Filters support prefix matching`?

Reply:
Ok, I see we have the same wording for other params that support filtering. I think its okay to keep the same wording here.